src/Sylius/Bundle/ProductBundle/Resources/config/serializer/Model.Product.yml

```diff
@@ -24,3 +24,4 @@ Sylius\Component\Product\Model\Product:
     virtual_properties:
         getVariants:
             serialized_name: variants
+            max_depth: 2
```

Comment:
Do we need this? As far as I know this will not allow for serialization of any variant collection like options, attributes as well as images.